src/pages/kubernetes.js

```diff
@@ -27,19 +27,24 @@ const KubernetesPage = () => {
                 <article class="p-8 h-full bg-ThemeCell tile">
                   <p>
                     The k8s-alpha CLI is deprecated. On <strong>March 31st, 2020</strong>, it will be <strong>removed</strong> from the linode-cli.
-                    After March 31, 2020, you will no longer be able to create or manage clusters created by the k8s-alpha CLI plugin.
+                    After March 31, 2020, you will no longer be able to create or manage clusters using the k8s-alpha CLI plugin.
                   </p><p>
-                    However, you will still be able to successfully manage your clusters using&nbsp;
+                    However, you will still be able to create and manage these clusters using&nbsp;
```

Comment:
Quick optional suggestion here, make the link text for this link "create and manage these clusters using Terraform", rather than just "Terraform", as it is in the [docs site warning](https://www.linode.com/docs/kubernetes/how-to-deploy-kubernetes-on-linode-with-k8s-alpha-cli/).

This provides a bigger target for users to click on.